docs/mp/server.adoc

```diff
@@ -338,6 +338,17 @@ server.static.classpath.welcome=resource.html
 # server.static.classpath.context=/static-cp
 ----
 
+=== Re-direct root using `server.base-path`
+To redirect requests for the root path (`/`) to another path you can use the `server.base-path` property:
+[source,yaml]
+----
+server:
+  base-path: /static/index.html
+----
+For any HTTP request for `/` this will return a 301 with the `Location:` header set to the value of `server.base-path`.
+This is often used with Static Content Support to serve a specific `index.html`  when `/` is requested.
+Note that this feature is not for setting a context root for applications. To configure alternate context roots see
+see xref:jaxrs/jaxrs-applications.adoc#_setting_application_path[Setting Application Path].
```

Comment:
Here is the link for `Setting Application Path`. This paragraph didn't exist and I've added it (copied from 4.x). As I can see `Setting Application Path` works for 3.x